### PR TITLE
Feat/vote buttons

### PR DESCRIPTION
**Pull request by jackieline**
functional vote buttons that update on client side and sends request to update the database and shows the data from database upon refresh

### CODE REVIEW

File: client/src/Events.jsx [hunk 1]
```diff
@@ -1,14 +1,53 @@
...
+		axios.put('/entry', {
+			timelineId: this.props.timelineId,
+			day: this.props.day.day,
+			eventId: this.props.event._id,
+			votes: this.state.votes,
+		})
+	}
+
+	updateVotes(e) {
+		console.log( this.props.event);
```
Comment (angelahuang89) [inline client/src/Events.jsx]: why you no take out console log

File: database/index.js [hunk 2]
```diff
@@ -34,13 +34,31 @@ const daySchema = mongoose.Schema({
 const Day = mongoose.model('Day', daySchema);
 const Event = mongoose.model('Event', eventSchema);
 
+const updateVotes = (timelineId, day, eventId, votes) => {
+  return Day.find({
+    day: day,
+    timelineId: timelineId, 
+    })
+  .then(results => {
+    console.log("this is the database results", results[0].events.id(eventId));
```
Comment (angelahuang89) [inline database/index.js]: please remove log fam, also the one on line 45

File: database/index.js [hunk 3]
```diff
@@ -34,13 +34,31 @@ const daySchema = mongoose.Schema({
...
+    var event = results[0].events.id(eventId)
```
Comment (angelahuang89) [inline database/index.js]: es6 please

File: database/index.js [hunk 4]
```diff
@@ -34,13 +34,31 @@ const daySchema = mongoose.Schema({
 const Day = mongoose.model('Day', daySchema);
 const Event = mongoose.model('Event', eventSchema);
 
+const updateVotes = (timelineId, day, eventId, votes) => {
+  return Day.find({
+    day: day,
+    timelineId: timelineId, 
+    })
+  .then(results => {
```
Comment (angelahuang89) [inline database/index.js:42]: nice promise

File: database/index.js [hunk 5]
```diff
@@ -34,13 +34,31 @@ const daySchema = mongoose.Schema({
...
+}
+
 const addNewTimeline = (timelineId, numberOfDays, timelineName) => {
   const timeline = [];
   for (let day = 1; day <= numberOfDays; day += 1) {
     const newDay = new Day({ day, timelineId, timelineName });
     timeline.push(newDay);
   }
-  return Promise.map(timeline, day => day.saveAsync());
+  return Promise.map(timeline, day => day.saveAsync())
```
Comment (angelahuang89) [inline database/index.js]: what happened to the winky boy?

File: server/index.js [hunk 6]
```diff
@@ -37,8 +37,10 @@ app.post('/entry', ({ body }, response) => {
 });
 
 app.put('/entry', (request, response) => {
-  // for editing a day entry in day model
-  response.send('for editing a day entry in day model');
+  db.updateVotes(request.body.timelineId, request.body.day, request.body.eventId, request.body.votes)
+    .then(() => response.sendStatus(200))
+    .catch(() => response.sendStatus(409));
+  // response.send('for editing a day entry in day model');
```
Comment (angelahuang89) [inline server/index.js]: this line can be removed :)

**Comment by andino93** [comment]
Hey so this is a PR to master branch. Can you PR to development instead

**Comment by andino93** [comment]
Looks good, can you PR for development when you get a chance? Thanks!

**Pull request closed without merging**